server/webapp/WEB-INF/rails.new/app/controllers/admin/templates_controller.rb

```diff
@@ -16,6 +16,8 @@
 
 class Admin::TemplatesController < AdminController
 
+  before_filter :check_admin_user_and_401, only: [:edit_permissions, :update_permissions, :new, :create]
+  before_filter :check_admin_or_template_admin_and_401, only: [:destroy, :update]
```

Comment:
Missed seeing this before, but `edit` action is now available for template-view-users. This needs to be fixed.